packages/app/src/pages/NewSpec.vue

```diff
@@ -49,11 +52,30 @@
             v-model="fileNameInput"
             class="flex-auto"
           />
-          <Button @Click="candidateClick(fileNameInput)">
+          <Button @click="candidateClick(fileNameInput)">
             Generate Spec
           </Button>
         </div>
       </template>
+      <template v-else-if="codeGenType === 'scaffoldIntegration'">
+        <div v-if="scaffoldIntegration.data.value">
+          <ul>
+            <li
+              v-for="generatedSpec of scaffoldIntegration.data.value.scaffoldIntegration"
+              :key="generatedSpec.fileParts.absolute"
+              data-cy="scaffold-integration-file"
+            >
+              <RouterLink
+                :key="generatedSpec.fileParts.absolute"
+                class="text-left"
+                :to="{ path: 'runner', query: { file: generatedSpec.fileParts.relative } }"
```

Comment:
Is this supposed to immediately execute the spec? I think the route has changed to something like `/spec?file=...`.

Reply:
This page was deleted, I'll integrate the scaffold integration into the new flow.